server/src/main/java/io/spine/server/aggregate/Aggregate.java

```diff
@@ -120,6 +120,16 @@
         extends CommandHandlingEntity<I, S, B>
         implements EventPlayer, EventReactor {
 
+    /**
+     * The count of events stored to the {@linkplain AggregateStorage storage} since the last
+     * snapshot.
+     *
+     * <p>This field is set in {@link #play(AggregateStateRecord)} and is effectively final.
+     *
+     * @see AggregateStorage#readEventCountAfterLastSnapshot(Object)
+     */
+    private int persistedEventCount = 0;
```

Comment:
To me, the name of the variable does not correspond to its description and usage.